Cauldron/Assets/Scripts/SO/Night events/NightEventProvider.cs

```diff
@@ -107,10 +111,14 @@ public NightEvent[] GetEvents(GameDataHandler game)
         {
             if (!PlayerPrefs.HasKey("FirstNight"))
```

Comment:
Let's add this key to PrefKeys static class

Reply:
Or actually it would be better to get rid of it and use the new round counter: round 0, day 1 (or 0)